### DISCUSSION TITLE
Help with TextrudeInteractive installation

### DISCUSSION THREAD

**Discussion by andbrs** [NeilMacMullen/Textrude#137] [Q&A]
Hello, how are you? I am trying to install this excellent tool, but the truth is that I am very new with this installation with the WebView2 runtime, could you list a step by step, because I do not know what to do after installing the browser plugin. thank you very much

**Comment by NeilMacMullen** [comment]
Hey @andbrs . You _should_ be be able to install and run TextrudeInteractive by following this sequence...

- If you don't already have .Net 5 installed go to https://dotnet.microsoft.com/download/dotnet/5.0 and download and run the .Net5 desktop runtime
- go to https://developer.microsoft.com/en-us/microsoft-edge/webview2/
- Download and run the "evergreen bootstrapper"
    - *This should result in the WebView runtime being installed*
- Download this zip file... https://github.com/NeilMacMullen/Textrude/releases/download/v1.6.0/Textrude.zip 
- Create a folder on your C drive called "Textrude" and copy the contents of the zip file into the local folder.
- Just double click on the TextrudeInteractive icon in c:\Textrude.

Does that make sense?  If you are running into specific problems let me know.  :-)

**Reply by NeilMacMullen** [reply]
I've updated the installation instructions [here](https://github.com/NeilMacMullen/Textrude/blob/main/doc/getBinaries.md).

**Reply by andbrs** [reply]
Thank you very much for the information, it is perfect. I'm going to start to understand the tool and understand the theme of templates
